staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion_test.go

```diff
@@ -87,8 +85,7 @@ func TestWebhookConverter(t *testing.T) {
 	etcd3watcher.TestOnlySetFatalOnDecodeError(false)
 	defer etcd3watcher.TestOnlySetFatalOnDecodeError(true)
 
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, apiextensionsfeatures.CustomResourceWebhookConversion, true)()
```

Comment:
why this switch? this leaks the feature gate change out of this test

Reply:
I didn't realize that. How does it get leaked?

Reply:
We should really start using server-local feature gates. This is insane.

@jpbetz our feature gates are global. My change of moving it to the flags (which is the right thing to do to avoid custom plumbing), lacks the defer code to set it back to normal.

Reply:
fixed.